spring-cloud-gcp-docs/src/main/asciidoc/memorystore.adoc

```diff
@@ -0,0 +1,21 @@
+== Cloud Memorystore for Redis
+
+https://cloud.google.com/memorystore/[Cloud Memorystore] for Redis provides a fully managed in-memory data store service
+built on scalable, secure, and highly available infrastructure managed by Google.
+Use Cloud Memorystore to build application caches that provides sub-millisecond data access.
```

Comment:
Did you copy this from somewhere? It sounds more like marketing material.

Reply:
@meltsufin 
Yes, it is an introductory text from Cloud Memorystore main page. Should I remove it?

Reply:
You can just say what Cloud Memorystore for Redis is -- something like " a managed Redis service on Google Cloud".